install_net.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Created Jun 19th 2019
+
+@author zackorenberg
+
+This script is designed to be a standalone installer for LabGui, to be used indefinitely until a valid replacement (via pip?) is created
```

Comment:
Your line is too long here. I suggest we use linters to make the code look uniform. I was thinking of giving `black` a try (check its [doc](https://black.readthedocs.io/en/stable/installation_and_usage.html#usage)).

When we implement the continuous integration (CI), I would also make use of the linters`flake8` and `pylint` to make the code look nice and uniform (it is actually important to make it more readable to others). `black` and `autopep8` can be used to make the changes directly in the files. This will be a little painful to implement, but then I will set up that one cannot do a merge if the linting tests do not pass -> it will force future addon to be pep8 compatible and coherently styled :)

Reply:
I do like nice looking code, but I really only like pep8 for comments. It is a pet peeve when a function's signature is on multiple lines, but I am willing to tolerate it for the other benefits of pep8. I am all for CI with linters

Reply:
I learned to like it (I had things I didn't like in PEP8, but the brain is flexible and `beautiful` is relative ;)), it is also nice to comment only one argument at the time while testing. The good thing with `black` is that it is consistent with itself and does the work automatically.